src/libCom/test/epicsTimerTest.cpp

```diff
@@ -27,6 +35,150 @@
 #define verify(exp) ((exp) ? (void)0 : \
     epicsAssert(__FILE__, __LINE__, #exp, epicsAssertAuthor))
 
+#if __cplusplus >= 201103L
+const double timeTolerance { 1e-3 + epicsThreadSleepQuantum() };
+
+class handler : public epicsTimerNotify
+{
+public:
+  handler(epicsTimerQueue& queue, std::function<void()> expireFctIn = [] {}) : timer(queue.createTimer()), expireFct(expireFctIn) {}
+  ~handler() { timer.destroy(); }
+  void start(double delay) {
+    startTime = epicsTime::getCurrent();
+    timer.start(*this, delay);
+  }
+  void cancel() { timer.cancel(); }
+  expireStatus expire(const epicsTime& currentTime) {
+    expireTime = epicsTime::getCurrent();
+    expireCount++;
+    expireFct();
+    return expireStatus(noRestart);
+  }
+  int getExpireCount() const { return expireCount; }
+  double getDelay() const { return expireTime - startTime; }
+private:
+  epicsTimer &timer;
+  int expireCount { 0 };
+  epicsTime startTime;
+  epicsTime expireTime;
+  std::function<void()> expireFct;
+};
+
+void verifyExpirationTime(double delta_t, double tolerance) {
```

Comment:
`delta_t` is slightly confusion to me.   In most c++ code I've seen, `*_t` is a type.

Reply:
:+1: Fixed.